### PR TITLE
Add example for taking screenshot of active window

### PR DESCRIPTION
**Pull request by CIAvash**
Using `swaymsg` and `jq`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ Select a region and screenshoot it:
 grim -g "$(slurp)"
 ```
 
+Select a region and screenshoot it, then edit it using `swappy`:
```
Comment (emersion) [inline README.md:31]: I'd rather not add this one, it's already in swappy's README.

Comment (CIAvash) [reply README.md:31]: If people don't know about swappy, then they can't read its README. But I'll remove it if that's your decision.

Comment (CIAvash) [reply README.md:31]: The thing is I recently switched to Wayland and Sway and I'm gradually finding new tools. And when I found grim, I first read its manual, and it did not have any examples. Later when I read grim's README, I thought I should have done this sooner.

File: README.md [hunk 2]
```diff
@@ -47,6 +47,13 @@ Grab a screenshot from the focused monitor under Sway, using `swaymsg` and
 grim -o $(swaymsg -t get_outputs | jq -r '.[] | select(.focused) | .name')
 ```
 
+Grab a screenshot from the focused window under Sway, using `swaymsg` and
+`jq`:
+
+```sh
+grim -g "$(swaymsg -t get_tree | jq -j '.. | select(.type?) | select(.focused==true).rect | "\(.x),\(.y) \(.width)x\(.height)"')"
```
Comment (emersion) [inline README.md:54]: Nit: `==true` is unnecessary

**Comment by emersion** [comment]
Thanks

**Pull request merged**